contracts/CommandBuilder.sol

```diff
@@ -4,6 +4,7 @@ uint8 constant VARIABLE_LENGTH = 0x80;
 uint8 constant INDEX_MASK = 0x7f;
 uint8 constant END_OF_ARGS = 0xff;
 uint8 constant USE_STATE = 0xfe;
+uint8 constant TUPLE_RETURN = 0xfd;
```

Comment:
We should probably clearly delineate which of these constants are bitmasks for the flags byte with some kind of prefix like `FLAG_`

Reply:
that TUPLE_RETURN value actually isn't used at all, remnant from the initial attempt, i'll remove